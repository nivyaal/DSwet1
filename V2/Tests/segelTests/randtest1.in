Init
AddArtist 94635 11
AddArtist 65885 -7
AddArtist -40040 16
AddArtist -81377 -50
AddArtist -45989 -22
AddArtist -97123 39
AddArtist -28392 17
AddArtist -31072 36
AddArtist -79047 1
AddArtist 23242 2
AddArtist 13573 36
AddArtist 47934 -46
AddArtist -15434 -4
AddArtist 38427 13
AddArtist -15573 -14
AddArtist 13061 -33
AddArtist -22319 -40
AddArtist -63261 -13
AddArtist 38410 12
AddArtist 76832 -27
AddArtist -92965 -37
AddArtist 23084 33
AddArtist 22971 1
AddArtist 31730 -32
AddArtist 48253 14
AddArtist -13044 18
AddArtist 55145 14
AddArtist -40847 38
AddArtist -47367 -12
AddArtist -53959 -31
AddArtist -46997 -27
AddArtist -31151 11
AddArtist 54895 13
AddArtist 57832 23
AddArtist 69384 -31
AddArtist -92965 -23
AddArtist -76016 3
AddArtist -95230 -38
AddArtist 62743 -33
AddArtist 32552 -20
AddArtist 50808 37
AddArtist 38370 8
AddArtist -95647 -1
AddArtist 12375 -25
AddArtist 14628 34
AddArtist 92286 -39
AddArtist -72432 -22
AddArtist 19972 32
AddArtist -69687 -43
AddArtist 20194 31
AddArtist -49630 -32
AddArtist -39313 -27
AddArtist -87327 -44
AddArtist 64526 47
AddArtist 79927 24
AddArtist 35052 -23
AddArtist 34955 39
AddArtist 15540 -27
AddArtist -12187 40
AddArtist 14373 -14
AddArtist 56169 4
AddArtist 71159 -22
AddArtist 16238 39
AddArtist -64818 -17
AddArtist -14478 2
AddArtist -37397 -46
AddArtist -16939 6
AddArtist 82543 47
AddArtist -92688 19
AddArtist -95456 23
AddArtist 40008 19
AddArtist -72888 -37
AddArtist -49212 -38
AddArtist -51412 -34
AddArtist 44084 11
AddArtist -60206 -4
AddArtist 74231 -50
AddArtist 86767 -24
AddArtist 81802 -50
AddArtist 60742 -30
AddArtist -63581 -8
AddArtist -24101 -26
AddArtist -68118 49
AddArtist 45218 -27
AddArtist 93407 -33
AddArtist -45876 -34
AddArtist -42454 -14
AddArtist -30008 8
AddArtist -75395 33
AddArtist 48281 -12
AddArtist 21651 -17
AddArtist 92239 -42
AddArtist -77098 -11
AddArtist 43491 16
AddArtist -52666 35
AddArtist 79516 -15
AddArtist -18227 -23
AddArtist -85111 -11
AddArtist -85627 38
AddArtist -85853 29
RemoveArtist 94635
RemoveArtist 65885
RemoveArtist -40040
RemoveArtist -81377
RemoveArtist -45989
RemoveArtist -97123
RemoveArtist -28392
RemoveArtist -31072
RemoveArtist -79047
RemoveArtist 23242
RemoveArtist 13573
RemoveArtist 47934
RemoveArtist -15434
RemoveArtist 38427
RemoveArtist -15573
RemoveArtist 13061
RemoveArtist -22319
RemoveArtist -63261
RemoveArtist 38410
RemoveArtist 76832
RemoveArtist -92965
RemoveArtist 23084
RemoveArtist 22971
RemoveArtist 31730
RemoveArtist 48253
RemoveArtist -13044
RemoveArtist 55145
RemoveArtist -40847
RemoveArtist -47367
RemoveArtist -53959
RemoveArtist -46997
RemoveArtist -31151
RemoveArtist 54895
RemoveArtist 57832
RemoveArtist 69384
RemoveArtist -92965
RemoveArtist -76016
RemoveArtist -95230
RemoveArtist 62743
RemoveArtist 32552
RemoveArtist 50808
RemoveArtist 38370
RemoveArtist -95647
RemoveArtist 12375
RemoveArtist 14628
RemoveArtist 92286
RemoveArtist -72432
RemoveArtist 19972
RemoveArtist -69687
RemoveArtist 20194
RemoveArtist -49630
RemoveArtist -39313
RemoveArtist -87327
RemoveArtist 64526
RemoveArtist 79927
RemoveArtist 35052
RemoveArtist 34955
RemoveArtist 15540
RemoveArtist -12187
RemoveArtist 14373
RemoveArtist 56169
RemoveArtist 71159
RemoveArtist 16238
RemoveArtist -64818
RemoveArtist -14478
RemoveArtist -37397
RemoveArtist -16939
RemoveArtist 82543
RemoveArtist -92688
RemoveArtist -95456
RemoveArtist 40008
RemoveArtist -72888
RemoveArtist -49212
RemoveArtist -51412
RemoveArtist 44084
RemoveArtist -60206
RemoveArtist 74231
RemoveArtist 86767
RemoveArtist 81802
RemoveArtist 60742
RemoveArtist -63581
RemoveArtist -24101
RemoveArtist -68118
RemoveArtist 45218
RemoveArtist 93407
RemoveArtist -45876
RemoveArtist -42454
RemoveArtist -30008
RemoveArtist -75395
RemoveArtist 48281
RemoveArtist 21651
RemoveArtist 92239
RemoveArtist -77098
RemoveArtist 43491
RemoveArtist -52666
RemoveArtist 79516
RemoveArtist -18227
RemoveArtist -85111
RemoveArtist -85627
RemoveArtist -85853
Quit
