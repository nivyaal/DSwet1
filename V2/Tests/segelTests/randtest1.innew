Init
AddCourse 94635 11
AddCourse 65885 -7
AddCourse -40040 16
AddCourse -81377 -50
AddCourse -45989 -22
AddCourse -97123 39
AddCourse -28392 17
AddCourse -31072 36
AddCourse -79047 1
AddCourse 23242 2
AddCourse 13573 36
AddCourse 47934 -46
AddCourse -15434 -4
AddCourse 38427 13
AddCourse -15573 -14
AddCourse 13061 -33
AddCourse -22319 -40
AddCourse -63261 -13
AddCourse 38410 12
AddCourse 76832 -27
AddCourse -92965 -37
AddCourse 23084 33
AddCourse 22971 1
AddCourse 31730 -32
AddCourse 48253 14
AddCourse -13044 18
AddCourse 55145 14
AddCourse -40847 38
AddCourse -47367 -12
AddCourse -53959 -31
AddCourse -46997 -27
AddCourse -31151 11
AddCourse 54895 13
AddCourse 57832 23
AddCourse 69384 -31
AddCourse -92965 -23
AddCourse -76016 3
AddCourse -95230 -38
AddCourse 62743 -33
AddCourse 32552 -20
AddCourse 50808 37
AddCourse 38370 8
AddCourse -95647 -1
AddCourse 12375 -25
AddCourse 14628 34
AddCourse 92286 -39
AddCourse -72432 -22
AddCourse 19972 32
AddCourse -69687 -43
AddCourse 20194 31
AddCourse -49630 -32
AddCourse -39313 -27
AddCourse -87327 -44
AddCourse 64526 47
AddCourse 79927 24
AddCourse 35052 -23
AddCourse 34955 39
AddCourse 15540 -27
AddCourse -12187 40
AddCourse 14373 -14
AddCourse 56169 4
AddCourse 71159 -22
AddCourse 16238 39
AddCourse -64818 -17
AddCourse -14478 2
AddCourse -37397 -46
AddCourse -16939 6
AddCourse 82543 47
AddCourse -92688 19
AddCourse -95456 23
AddCourse 40008 19
AddCourse -72888 -37
AddCourse -49212 -38
AddCourse -51412 -34
AddCourse 44084 11
AddCourse -60206 -4
AddCourse 74231 -50
AddCourse 86767 -24
AddCourse 81802 -50
AddCourse 60742 -30
AddCourse -63581 -8
AddCourse -24101 -26
AddCourse -68118 49
AddCourse 45218 -27
AddCourse 93407 -33
AddCourse -45876 -34
AddCourse -42454 -14
AddCourse -30008 8
AddCourse -75395 33
AddCourse 48281 -12
AddCourse 21651 -17
AddCourse 92239 -42
AddCourse -77098 -11
AddCourse 43491 16
AddCourse -52666 35
AddCourse 79516 -15
AddCourse -18227 -23
AddCourse -85111 -11
AddCourse -85627 38
AddCourse -85853 29
RemoveCourse 94635
RemoveCourse 65885
RemoveCourse -40040
RemoveCourse -81377
RemoveCourse -45989
RemoveCourse -97123
RemoveCourse -28392
RemoveCourse -31072
RemoveCourse -79047
RemoveCourse 23242
RemoveCourse 13573
RemoveCourse 47934
RemoveCourse -15434
RemoveCourse 38427
RemoveCourse -15573
RemoveCourse 13061
RemoveCourse -22319
RemoveCourse -63261
RemoveCourse 38410
RemoveCourse 76832
RemoveCourse -92965
RemoveCourse 23084
RemoveCourse 22971
RemoveCourse 31730
RemoveCourse 48253
RemoveCourse -13044
RemoveCourse 55145
RemoveCourse -40847
RemoveCourse -47367
RemoveCourse -53959
RemoveCourse -46997
RemoveCourse -31151
RemoveCourse 54895
RemoveCourse 57832
RemoveCourse 69384
RemoveCourse -92965
RemoveCourse -76016
RemoveCourse -95230
RemoveCourse 62743
RemoveCourse 32552
RemoveCourse 50808
RemoveCourse 38370
RemoveCourse -95647
RemoveCourse 12375
RemoveCourse 14628
RemoveCourse 92286
RemoveCourse -72432
RemoveCourse 19972
RemoveCourse -69687
RemoveCourse 20194
RemoveCourse -49630
RemoveCourse -39313
RemoveCourse -87327
RemoveCourse 64526
RemoveCourse 79927
RemoveCourse 35052
RemoveCourse 34955
RemoveCourse 15540
RemoveCourse -12187
RemoveCourse 14373
RemoveCourse 56169
RemoveCourse 71159
RemoveCourse 16238
RemoveCourse -64818
RemoveCourse -14478
RemoveCourse -37397
RemoveCourse -16939
RemoveCourse 82543
RemoveCourse -92688
RemoveCourse -95456
RemoveCourse 40008
RemoveCourse -72888
RemoveCourse -49212
RemoveCourse -51412
RemoveCourse 44084
RemoveCourse -60206
RemoveCourse 74231
RemoveCourse 86767
RemoveCourse 81802
RemoveCourse 60742
RemoveCourse -63581
RemoveCourse -24101
RemoveCourse -68118
RemoveCourse 45218
RemoveCourse 93407
RemoveCourse -45876
RemoveCourse -42454
RemoveCourse -30008
RemoveCourse -75395
RemoveCourse 48281
RemoveCourse 21651
RemoveCourse 92239
RemoveCourse -77098
RemoveCourse 43491
RemoveCourse -52666
RemoveCourse 79516
RemoveCourse -18227
RemoveCourse -85111
RemoveCourse -85627
RemoveCourse -85853
Quit
