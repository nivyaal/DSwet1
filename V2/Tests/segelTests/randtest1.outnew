init done.
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: SUCCESS
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
AddCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: FAILURE
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: SUCCESS
RemoveCourse: INVALID_INPUT
RemoveCourse: FAILURE
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
RemoveCourse: INVALID_INPUT
quit done.
