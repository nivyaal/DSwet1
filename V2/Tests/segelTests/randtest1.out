init done.
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: SUCCESS
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
AddArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: FAILURE
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: SUCCESS
RemoveArtist: INVALID_INPUT
RemoveArtist: FAILURE
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
RemoveArtist: INVALID_INPUT
quit done.
